esmvaltool/recipes/recipe_tebaldi21esd.yml

```diff
@@ -0,0 +1,7993 @@
+# ESMValTool
+# recipe_tebaldi21esd.yml
+---
+documentation:
+  title: Tebaldi et al.,ESD, 2021
```

Comment:
Would it be an option to use the title of the paper instead, e.g. "Climate model projections from the ScenarioMIP of CMIP6". This would read better especially in the html reports, see [here](https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0/)

Reply:
Yes sure we can use this title instead. That is fine by me !